internal/client/github.go

```diff
@@ -56,6 +56,36 @@ func NewGitHub(ctx *context.Context, token string) (Client, error) {
 	return &githubClient{client: client}, nil
 }
 
+// CloseMilestone closes a given milestone.
+func (c *githubClient) CloseMilestone(ctx *context.Context, repo Repo, title string) error {
+	milestone, err := c.getMilestoneByTitle(ctx, repo, title)
+
+	if err != nil {
+		return err
+	}
+
+	if milestone == nil {
+		return fmt.Errorf("no open milestone matching: %s", title)
+	}
+
+	closedState := "closed"
+	milestone.State = &closedState
+
+	_, _, err = c.client.Issues.EditMilestone(
+		ctx,
+		repo.Owner,
+		repo.Name,
+		*milestone.Number,
+		milestone,
+	)
+
+	if err != nil {
```

Comment:
could be just `return err`

Reply:
Updated in https://github.com/goreleaser/goreleaser/pull/1657/commits/5aea6dd9fed3c81f9878bae771456b73722ba4a7 👍